src/main/java-templates/org/postgresql/Driver.java

```diff
@@ -411,7 +411,7 @@ public Connection getResult(long timeout) throws SQLException {
      * @throws SQLException if the connection could not be made
      */
     private static Connection makeConnection(String url, Properties props) throws SQLException {
-        return new @JDBCCONNECTCLASS@(hostSpecs(props),
+        return new ${JDBCCONNECTCLASS}(hostSpecs(props),
```

Comment:
I should have published the ground rules earlier.

1) The priority is not to build to the latest Java version. The priority is
to make sure the largest reasonable population has latest drivers for any
version of PostgreSQL currently supported. What this means practically is
that we have to have jars for 1.6.1,7, 1.8, etc

2) We cannot change the code other than re-arrange the source. No surprises
for anyone.


If this can't be accomplished with Maven then IMO this is a non-starter.
Building with ant is not that much different.

git checkout ...
ant jar

Dave
r

On 14 June 2015 at 16:07, Stephen Nelson <notifications@github.com> wrote:

> In src/main/java-templates/org/postgresql/Driver.java
> <https://github.com/pgjdbc/pgjdbc/pull/322#discussion_r32383661>:
>
> > @@ -411,7 +411,7 @@ public Connection getResult(long timeout) throws SQLException {
> >       * @throws SQLException if the connection could not be made
> >       */
> >      private static Connection makeConnection(String url, Properties props) throws SQLException {
> > -        return new @JDBCCONNECTCLASS@(hostSpecs(props),
> > +        return new ${JDBCCONNECTCLASS}(hostSpecs(props),
>
> @vlsi <https://github.com/vlsi> I'm drafting a mailing list message right
> now :-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pgjdbc/pgjdbc/pull/322/files#r32383661>.
>